sklearn/ensemble/_hist_gradient_boosting/predictor.py

```diff
@@ -78,3 +79,20 @@ def predict_binned(self, X):
         out = np.empty(X.shape[0], dtype=Y_DTYPE)
         _predict_from_binned_data(self.nodes, X, out)
         return out
+
+    def compute_partial_dependence(self, grid, target_features, out):
```

Comment:
I would rather make this a private method even if `TreePredictor` is not considered part of the public API.

Reply:
Actually the same method is already present in the tree instances of the traditional GBRT. So it makes the code simpler if we reuse this convention here.